server/server.go

```diff
@@ -83,7 +83,7 @@ type Config struct {
 
 	RegisterInstrumentation  bool `yaml:"register_instrumentation"`
 	ExcludeRequestInLog      bool `yaml:"-"`
-	DisableRequestSuccessLog bool `yaml:"-"`
+	DisableRequestSuccessLog bool `yaml:"disable_request_success_log"`
```

Comment:
Somehow my comment here got lost. 
I think if you do this without adding a flag it shows up in docs with no explanation, which is undesirable.
[Example](https://github.com/grafana/mimir/pull/4922/files#diff-809450d2d139bdb2005dd008d79c44bbad79aa5f76dacd68a5b46c7f3fe669c6R573)
I would prefer you expose in your own program, and leave it unexposed in weaveworks/common, as ExcludeRequestInLog is.

Reply:
aháaaa I definitely overlooked that, great call. That for sure works, thank you.

Reply:
Now I understand the context a bit better, my preference is to leave this as `"-"`.
You can add any config or flags you want in your downstream application.

Reply:
Addressed by https://github.com/weaveworks/common/pull/284/commits/ed6819b7603030a18fc24bec7dbfbce39f318a05.